org-roam.el

```diff
@@ -798,17 +608,17 @@ Otherwise, behave as if called interactively."
     (dolist (buf (org-roam--get-roam-buffers))
       (with-current-buffer buf
         (org-link-set-parameters "file" :face 'org-link)
-        (remove-hook 'post-command-hook #'org-roam--maybe-update-buffer t)
+        (remove-hook 'post-command-hook #'org-roam-buffer--update-maybe t)
         (remove-hook 'after-save-hook #'org-roam-db--update-file t))))))
 
 (defun org-roam--find-file-hook-function ()
   "Called by `find-file-hook' when mode `org-roam-mode' is on."
   (when (org-roam--org-roam-file-p)
     (setq org-roam-last-window (get-buffer-window))
-    (add-hook 'post-command-hook #'org-roam--maybe-update-buffer nil t)
+    (add-hook 'post-command-hook #'org-roam-buffer--update-maybe nil t)
     (add-hook 'after-save-hook #'org-roam-db--update-file nil t)
     (org-link-set-parameters "file" :face 'org-roam--roam-link-face)
-    (org-roam--maybe-update-buffer :redisplay nil)))
+    (org-roam-buffer--update-maybe :redisplay t)))
```

Comment:
the value here changed, but afaict it shouldn't affect behaviour, since find-file should cause an update anyway.

Reply:
I believe I changed this due to another, unrelated issue I was investigating. Happy to change it back if you like.